src/js/App/CrossRequestNotifier/CrossRequestNotifier.tsx

```diff
@@ -1,52 +1,47 @@
 import React, { useReducer } from 'react';
-import PropTypes from 'prop-types';
-import Portal from '@redhat-cloud-services/frontend-components-notifications/Portal';
+import Portal, { PortalNotificationConfig } from '@redhat-cloud-services/frontend-components-notifications/Portal';
+import { AlertVariant } from '@patternfly/react-core';
 import { ACCOUNT_REQUEST_TIMEOUT, ACTIVE_ACCOUNT_SWITCH_NOTIFICATION } from '../../consts';
 import useAccessRequestNotifier from '../../utils/useAccessRequestNotifier';
 import ChromeLink from '../Sidenav/Navigation/ChromeLink';
 
 const ACCOUNT_CHANGE_ID = 'account_change';
 const ACCOUNT_TIMEOUT_ID = 'account_timeout';
 
-const accountSwitchNotification = {
+const accountSwitchNotification: PortalNotificationConfig = {
   id: ACCOUNT_CHANGE_ID,
   title: 'View has changed',
   description: 'You are now viewing console.redhat.com as a customer, and will be interacting with real customer data.',
   variant: 'info',
 };
 
-const defaultNotificationConfig = {
+const defaultNotificationConfig: Omit<PortalNotificationConfig, 'id'> = {
   variant: 'info',
   dismissable: true,
   title: 'You have a new access request that needs your review',
 };
 
-const createAccoutTimeoutNotification = (accountId) => ({
-  variant: 'danger',
+const createAccoutTimeoutNotification = (accountId: string) => ({
+  variant: AlertVariant.danger,
   id: ACCOUNT_TIMEOUT_ID,
   title: `You no longer have access to account ${accountId}.`,
   autoDismiss: false,
 });
 
-const DescriptionComponent = ({ id, markRead }) => (
+const DescriptionComponent = ({ id, markRead }: { id: string; markRead: (id: string) => void }) => (
   <span onClick={() => markRead(id)}>
     <ChromeLink href={id === 'mark-all' ? '/settings/rbac/access-requests' : `/settings/rbac/access-requests/${id}`} appId="rbac">
       View request
     </ChromeLink>
   </span>
 );
 
-DescriptionComponent.propTypes = {
-  id: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.string), PropTypes.string]).isRequired,
-  markRead: PropTypes.func.isRequired,
-};
-
 const CrossRequestNotifier = () => {
-  const [, forceRender] = useReducer((state = false) => !state);
+  const [, forceRender] = useReducer((state) => state + 1, 0);
```

Comment:
Any reason for the `+1`?

Reply:
Yes. Actually using the `true/false` can sometimes run through multiple updates and will not resolve in a forced re-render. We had this issue in DDF and figured we can't use a simple boolean flip. Hence the counter.

Reply:
Okay, works for me